### PR TITLE
[release/2.1.8xx] Merge from 2.1.5xx and 2.1.6xx

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: build/publish/PublishNupkgToBlobFeed.targets [hunk 1]
```diff
@@ -13,6 +13,17 @@
...
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.$(Architecture).*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' " />
+      <VSNupkgsForPublishing
+        Include="$(PackagesDirectory)/VS.Redist.Common.Net.Core.SDK.MSBuildExtensions.*.nupkg"
+        Condition=" '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' " />
+
+      <NupkgsForPublishing Include="@(VSNupkgsForPublishing)">
+        <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
+      </NupkgsForPublishing>
```
Comment (mmitche) [inline build/publish/PublishNupkgToBlobFeed.targets:26]: @wli3 @dsplaisted Any reason these changes were not in 2.1.8xx?

Comment (dsplaisted) [reply build/publish/PublishNupkgToBlobFeed.targets:26]: Perhaps because we don't insert 2.1.8xx into VS at all.

Comment (mmitche) [reply build/publish/PublishNupkgToBlobFeed.targets:26]: Ahh. No harm in this change though. Just makes it go to the blob feed.

**Comment by mmitche** [comment]
Unrelated failures, merging to unblock

**Pull request merged**